setup.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+#
+# Copyright 2013 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Setup configuration."""
+
+try:
+    import setuptools
+except ImportError:
+    from ez_setup import use_setuptools
+    use_setuptools()
+    import setuptools
+
+
+setuptools.setup(
+    name='pyu2f',
+    version='HEAD',
+    description='U2F host library for interacting with a U2F device over USB.',
+    long_description='pyu2f is a python based U2F host library for Linux, '
+                     'Windows, and MacOS. It provides functionality for '
+                     'interacting with a U2F device over USB.',
+    url='https://github.com/google/pyu2f/',
+    author='Google Inc.',
+    author_email='gnubby-team@google.com',
```

Comment:
Lets make this pyu2f-team@google.com, which I just created with relevant membership